client/js/lounge.js

```diff
@@ -227,6 +227,10 @@ $(function() {
 		var chan = chat.find(target);
 		var msg;
 
+		if (highlights.some((h) => data.msg.text.indexOf(h) > -1)) {
```

Comment:
I will remove it for now, but I would really want to add babel at some point. We can use arrow functions on the server, but not the client, and it seems a massive shame to leave out such nice development tools just because rubbish browsers.

I have a little bit of an obsession with arrow functions just now.